engine/jobmaster/dm/runtime/worker_status.go

```diff
@@ -25,31 +25,34 @@ import (
 var HeartbeatInterval = 3 * time.Second
 
 // WorkerStage represents the stage of a worker.
-//          ,──────────────.      ,────────────.      ,─────────────.     ,──────────────.
-//          │WorkerCreating│      │WorkerOnline│      │WorkerOffline│     │WorkerFinished│
-//          `──────┬───────'      `─────┬──────'      `──────┬──────'     `──────┬───────'
-//                 │                    │                    │                   │
-//   CreateWorker  │                    │                    │                   │
+//
```

Comment:
This change seems strange.

Reply:
I don't know why this file has been changed, I will revert it as soon as possible.